lsmb-request.pl

```diff
@@ -19,6 +19,7 @@ =head1 COPYRIGHT
 =cut
 
 package LedgerSMB::Handler;
+use lib '.';
```

Comment:
I don't have a problem with a block of code similar to that.  However there are a couple costs to think about:

1.  That block won't work as such.  You can't really use variables with use because use is compile time. while your path operations are run-time
2.  Working around 1  above means significant effort, a begin block, and is something far more brittle.

I guess the question is given that this is only going in the http request handlers, what would be gained by that complexity.  I do agree it would not be a great idea to put in command line tools but then you would not run a command line tool in that way from just anywhere, so presumably we'd make findbin work (and for some reason on Debian it is *not* working so this is a workaround).